judo-ui-react/src/main/java/hu/blackbelt/judo/ui/generator/react/UiI18NHelper.java

```diff
@@ -316,4 +322,106 @@ private static Boolean keepTranslationKey(String key) {
         Matcher m = TRANSLATION_KEYS_TO_SKIP.matcher(key);
         return !m.matches();
     }
+
+    public static String magicTranslateFlex(Flex flex) {
```

Comment:
I find function names that start with "magic" funny, but I believe they should not be left as they currently are:D